packages/terra-menu/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
```

Comment:
this `changelog` is not required as we are not making any changes on terra-menu component.

Reply:
Removed